src/test/java/seedu/ccacommander/storage/StorageManagerTest.java

```diff
@@ -24,7 +24,7 @@ public class StorageManagerTest {
 
     @BeforeEach
     public void setUp() {
-        JsonAddressBookStorage addressBookStorage = new JsonAddressBookStorage(getTempFilePath("ab"));
+        JsonCcaCommanderStorage addressBookStorage = new JsonCcaCommanderStorage(getTempFilePath("ab"));
         JsonUserPrefsStorage userPrefsStorage = new JsonUserPrefsStorage(getTempFilePath("prefs"));
         storageManager = new StorageManager(addressBookStorage, userPrefsStorage);
```

Comment:
Corresponding Suggestion
```suggestion
        storageManager = new StorageManager(ccaCommanderStorage, userPrefsStorage);
```

Reply:
resolved